src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Backup/BackupStep.cs

```diff
@@ -132,16 +135,19 @@ protected override async Task<UpgradeStepApplyResult> ApplyImplAsync(IUpgradeCon
 
         public override UpgradeStepInitializeResult Reset()
         {
-            _backupPath = null;
+            _properties.RemovePropertyValue(BackupPropertyName);
```

Comment:
If you want it to persist, you can remove the whole method. This method is (usually) called when you switch to a new project

Reply:
Ok, I misunderstood the purpose.